multi-arch-builders/provisioning/aarch64/main.tf

```diff
@@ -0,0 +1,51 @@
+terraform {
+  required_providers {
+    ct = {
+      source  = "poseidon/ct"
+      version = "0.13.0"
+    }
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+    http = {
+      source  = "hashicorp/http"
+      version = "2.1.0"
+    }
+  }
+}
+
+provider "aws" {}
+provider "ct" {}
+provider "http" {}
+
+variable "aws_ami" {
+ type        = string
+}
+
+# Get ignition created for the multiarch builder
+resource "null_resource" "butane" {
+  provisioner "local-exec" {
+    command = "bash -x ./butane.sh" 
+  }
+}
+
+resource "aws_instance" "coreos-multiarch-builder-aarch64" {
+  tags = {
+    Name = "coreos-aarch64-builder-${formatdate("YYYYMMDD", timestamp())}"
+  }
+  ami           = var.aws_ami
+  user_data     = file("coreos-aarch64-builder.ign")
+  instance_type = "m6g.metal"
+# TODO: fix AMI permission to create security groups
+  vpc_security_group_ids = ["sg-06b1c86b7f2808ec5"] 
+  subnet_id              = var.aws_subnet_internal
+  root_block_device {
+      volume_size = "200"
+      volume_type = "gp3"
+  }
+}
+
+output "instance_ip_addr" {
+  value = aws_instance.coreos-multiarch-builder-aarch64.private_ip
```

Comment:
`private_ip` - so this instance can only be reached via a jump host?

Reply:
It is has a vpc and the internal subnet. So, no need, if you are in RH networking, you are able to access it.

Reply:
That's probably another if case we will need for rhcos/fcos. Fcos does have public ip